src/XeroPHP/Remote/Query.php

```diff
@@ -20,6 +25,16 @@ class Query {
     private $page;
     private $offset;
 
+    /**
+     * @var bool|null
+     */
+    private $includeArchived = null;
```

Comment:
@calcinai if it's null then it won't be added to query at all. If it's true or false then it's added to query as true or false.